### PR TITLE
Event Subscriptions

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Description**

Changes proposed in this pull request:

- Add Event Subscriptions depending on config variable

**Related issue(s)**
[Kyma](https://github.com/kyma-project/kyma/pull/10186)
[Bump](https://github.com/kyma-project/kyma/pull/10204)

### CODE REVIEW

File: core-ui/src/shared/components/EventSubscriptions/CreateSubscriptionModal.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+    try {
+      await createEventSubscription({ variables });
+      notification.notifySuccess({
+        content: EVENT_SUBSCRIPTION_PANEL.ADD_MODAL.NOTIFICATION.SUCCESS,
+      });
+    } catch (e) {
+      console.warn(e);
+      notification.notifyError({
+        content: `${EVENT_SUBSCRIPTION_PANEL.ADD_MODAL.NOTIFICATION.ERROR} ${e.message}`,
```
Comment (parostatkiem) [inline core-ui/src/shared/components/EventSubscriptions/CreateSubscriptionModal.js:72]: How about using `title` and `content` props? I think `title` also would be better in line 67.

**Review by parostatkiem — CHANGES_REQUESTED** [review]

File: core-ui/src/gql/queries.js [hunk 2]
```diff
@@ -387,3 +387,23 @@ export const GET_SECRET_DETAILS = gql`
...
+
+export const GET_EVENT_SUBSCRIPTIONS = gql`
+  query eventSubscriptions($ownerName: String!, $namespace: String!) {
+    eventSubscriptions(ownerName: $ownerName, namespace: $namespace) {
+      name
+      namespace
+      spec {
+        filter {
+          filters {
+            eventType {
```
[the remark this answers is not in the document]
Comment (akucharska) [reply core-ui/src/gql/queries.js:399]: The naming is based on tunas structure

**Review by parostatkiem — DISMISSED** [review]

File: core-ui/src/components/Lambdas/LambdaDetails/LambdaDetails.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ import LambdaDetailsHeader from './LambdaDetailsHeader/LambdaDetailsHeader';
 import CodeTab from './Tabs/Code/CodeTab';
 import ResourceManagementTab from './Tabs/ResourceManagement/ResourceManagementTab';
 
-import EventTriggersWrapper from './Tabs/Configuration/EventTriggers/EventTriggersWrapper';
+import EventTriggersWrapper from './Tabs/Configuration/EventTriggers/LambdaEventsWrapper';
```
Comment (dariadomagala) [inline core-ui/src/components/Lambdas/LambdaDetails/LambdaDetails.js:9]: Shouldn't that be
```suggestion
import LambdaEventsWrapper from './Tabs/Configuration/EventTriggers/LambdaEventsWrapper';
```
?

File: core-ui/src/components/Services/ServiceDetails/ServiceDetails.js [hunk 4]
```diff
@@ -10,7 +10,7 @@ import {
 } from 'react-shared';
 import ServiceDetailsHeader from './ServiceDetailsHeader/ServiceDetailsHeader';
 import ServiceApiRules from './ServiceApiRules/ServiceApiRules';
-import ServiceEventTriggers from './ServiceEventTriggers/ServiceEventTriggers';
+import ServiceEventTriggers from './ServiceEventTriggers/ServiceEventsWrapper';
```
Comment (dariadomagala) [inline core-ui/src/components/Services/ServiceDetails/ServiceDetails.js:13]: Same here 
```suggestion
import ServiceEventsWrapper from './ServiceEventTriggers/ServiceEventsWrapper';
```

**Review by dariadomagala — APPROVED** [review]

**Review by aerfio — APPROVED** [review]

**Pull request merged**